pkg/controller/config/config_controller_test.go

```diff
@@ -140,7 +150,10 @@ func TestReconcile(t *testing.T) {
 		return nil
 	}, timeout).Should(gomega.BeNil())
 
-	cancel()
+	testMgrStopped()
```

Comment:
why do we need to call this twice?

Reply:
The defer guarantees that the test manager gets stopped if the test exits early, so that it doesn't interfere with other tests. The in-band call stops the test manager so that we can run cleanup routines without reconcilers re-adding the cleaned state.

Note that the actual function is only run once due to sync.Once